book/src/user_manual/crypto-subtle.md

```diff
@@ -0,0 +1,41 @@
+# crypto-subtle feature
+
+This feature of the OpenMLS crate allows importing and exporting private signature keys that can be used with credentials.
+
+⚠️ Note that no checks are performed on the keys, use this feature at your own risk.
```

Comment:
Just out of an abundance of caution, can you add a note here that essentially says: If you create a credential from an existing key or if you export key material then you are responsible for the deletion of that key. If that key is kept around outside of OpenMLS, updating a leaf is not going to get you FS/PCS.